.github/workflows/versions-check.yml

```diff
@@ -386,7 +386,7 @@ jobs:
 
     strategy:
       matrix:
-        test-version: [ 1.6.21, 1.7.21, 1.8.22, 1.9.22 ]
```

Comment:
I get it that Kotlin has a "you better upgrade or good luck" versioning scheme but there will be people that are stuck on older versions. I'd suggest keeping 1.9 and 2.0 for now.

Reply:
Okay. From looking at the Kotlin site, I would be extremely uncomfortable running an older version as there seems to be basically no support other than upgrading (no new patches on old minor versions seem to ever happen). But to each their own :)